app/weddingapp/models.py

```diff
@@ -1,3 +1,60 @@
-# from django.db import models
+"""
+Database models
+"""
+from django.db import models
+from django.contrib.auth.models import (
+    AbstractBaseUser,
+    BaseUserManager,
+    PermissionsMixin
+)
 
-# Create your models here.
+
+class UserManager(BaseUserManager):
```

Comment:
I think it's better put this class in another file called "managers.py", since it's not a db model and for a better organization of the code.